printer_mods/MakeALeft/Switchwire_Front_Grill/readme.md

```diff
@@ -0,0 +1,2 @@
+# Why?

```

Comment:
Is the install process the exact same as for a regular switchwire? If not I would add some more detailed instructions.